bin/esvalidate.js

```diff
@@ -101,17 +112,16 @@ process.argv.splice(2).forEach(function (entry) {
             console.log('Error: unknown report format ' + options.format + '.');
             process.exit(1);
         }
-    } else if (entry.slice(0, 2) === '--') {
+    } else if (entry === '--') {
+        forceFile = true;
```

Comment:
Likewise here, although I suppose this would be little more than `forceFileIndex = fnames.length`. Let me know if you want the robustness.